### ISSUE TITLE
for OBO child -- id: About your home continued

### ISSUE THREAD

**Issue by VTskier** [VTskier/docassemble-RFApackage#344]
also offer the defendant's home address as a choice.

**Comment by tobyfey** [comment]
I don't believe this question was asked and I don't see in the interview order where it would be

**Comment by VTskier** [comment]
These are the parts of the interview order where it is asked:

    if users1_no_shelter: 
      rent_or_own


    if children1_no_shelter:
      rent_or_own

**Comment by tobyfey** [comment]
<img width="904" height="342" alt="Image" src="https://github.com/user-attachments/assets/19e04fa7-cbbd-4348-9d3b-ad08771ae63b" />
